tests/airflow/dags/operators/test_connection.py

```diff
@@ -3,7 +3,7 @@
 from sqlalchemy import select
 from sqlalchemy.orm.session import Session
 
-from src.airflow.dags.operators.connection import (
+from operators.connection import (
```

Comment:
one of the test does not work beacause of this. Missing import ?

Reply:
I had problems with the imports. I fixed it by using composer local dev and changing this import.